themes/dark_spinel.json

```diff
@@ -91,7 +91,6 @@
     {
       "name": "Punctuation, function and method calls",
       "scope": [
-        "punctuation",
```

Comment:
Let's not remove punctuation from here. Most of the time, we want punctuation to be the basic foreground colour. Instead, we can add more specific punctuation scopes to the colours you just added for the diffs.


The easiest way of discovering the scopes is by opening the command palette (CMD + SHIFT + P) and using `Inspect editor tokens and scopes`. If you try that in the patch, you'll discover the specific scopes you want to include are:
- `punctuation.definition.deleted.diff` (red)
- `punctuation.definition.inserted.diff` (green)
- `punctuation.definition.to-file.diff` (blue)

Not sure if there's any punctuation that would be associated with the orange. These three seem to do the trick

Reply:
Good point, I wasn't sure what was the best strategy. I updated the commit with the required scopes.